website/src/main/resources/microsite/plugins/RuleRefTag.rb

```diff
@@ -0,0 +1,23 @@
+module Jekyll
+  class RuleRefTag < Liquid::Tag
+
+    def initialize(tag_name, docTitle, tokens)
+      super
+      @docTitle, @hash = docTitle.split(',').map(&:strip)
+    end
+
+    def render(context)
+      rules = context['site']['data']['rules']['rules']
+      rule = rules.detect { |r| r == @docTitle }
+      if rule.nil? then
+        raise ArgumentError, "Could not find a rule '#{@docTitle}' in file rules.yml"
+      else
+        name = @docTitle
```

Comment:
`name` is a bit redundant here. I would just use `@docTitle` directly

Reply:
Fixed. Renamed to `@rule`, no idea what the `@` is for.